lisa/util/shell.py

```diff
@@ -57,6 +58,40 @@ def wait_tcp_port_ready(
     return is_ready, result
 
 
+def wait_for_ssh_to_be_ready(
```

Comment:
It's similar usage like try_connect. Can you refine that method, to make it default wait for 5 seconds, but can be extend by the parameter? So, by default, it's a short timeout, and assume the service is ready. But it can be used in case the server is booting up, and ssh is unstable.

Reply:
I mean, can you just use the `try_connect`? If any logic needs to be merged into the `try_connect`, it's ok to go with.

Reply:
I got confused because there were comments present on the ssh_ready function. Thanks for clarifying!